extra_data/reader.py

```diff
@@ -95,7 +96,7 @@ def __init__(self, filename, _cache_info=None):
 
             # Close the file again - crude way to avoid hitting the limit of
             # open files, which is only 1024 by default.
-            self.close()
```

Comment:
I agree that this can be removed - let's get rid of it entirely,  along with the comment.

Reply:
accepted